### PR TITLE
feat: add settings

### LINKED ISSUE #13: add settings
- Waves - Flux
- Albedo
- Greenhouse Effect: CO2, CH4, H2O
- Feedbacks: Ice cover change, Vapeur d'eau, Fonte du permafrost

They will be saved in redux but have no impact on the canvas/simulation.

### PR DESCRIPTION
**Pull request by pyphilia**
Screenshot 2021-05-17 at 14 29 53

Compared to the mockup, there is the play/pause button missing (part of issue #10) and "values in X" which I am not sure how we will handle.

close #13

### CODE REVIEW

File: src/components/common/AlbedoSlider.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  const dispatch = useDispatch();
+
+  const onChange = (e, v) => {
+    dispatch(setAlbedo(v));
+  };
+
+  return (
+    <SliderWithLabel
+      text={t('Albedo')}
+      max={100}
```
Comment (hagoptaminian) [inline src/components/common/AlbedoSlider.js:19]: maybe outsource to `constants.js`?

File: src/components/common/GreenhouseEffectsSettings.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+import { useTranslation } from 'react-i18next';
+import { useDispatch, useSelector } from 'react-redux';
+import SliderWithLabel from './SliderWithLabel';
+import { setGreenhouseGasesValues } from '../../actions';
+
+const GreenhouseEffectsSettings = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const values = useSelector(({ lab }) => lab.greenhouseGasesValues);
+  const { ch4, co2, h2o } = values;
```
Comment (hagoptaminian) [inline src/components/common/GreenhouseEffectsSettings.js:12]: can we use _methane_, _carbonDioxide_, _water_? feel like it's easier/safer than having numbers in the variable names...

File: src/components/common/GreenhouseEffectsSettings.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+  const onChange = (value, key) => {
+    dispatch(setGreenhouseGasesValues({ [key]: value }));
+  };
+
+  return (
+    <>
+      <Typography variant="h6">{t('Greenhouse Effects')}</Typography>
+      <SliderWithLabel
+        text={t('CH_4 (ppm)', { escapeInterpolation: true })}
+        max={10}
```
Comment (hagoptaminian) [inline src/components/common/GreenhouseEffectsSettings.js:23]: maybe move to `constants.js`... (with other constants in this file)

File: src/components/common/RadiationModeSwitch.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+import SwitchWithTwoLabels from './SwitchWithTwoLabels';
+
+const RadiationModeSwitch = () => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const radiationMode = useSelector(({ lab }) => lab.radiationMode);
+
+  const rightLabel = (
+    <div
+      // eslint-disable-next-line react/no-danger
```
Comment (hagoptaminian) [inline src/components/common/RadiationModeSwitch.js:15]: would help to have a comment explaining why we do this.

File: src/components/common/GreenhouseEffectsSettings.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+        value={ch4}
+        onChange={(e, v) => onChange(v, 'ch4')}
+      />
+      <SliderWithLabel
+        text={t('H_2O (ppm)', { escapeInterpolation: true })}
```
Comment (hagoptaminian) [inline src/components/common/GreenhouseEffectsSettings.js:28]: did not know that _ makes the text subscript...!

**Review by hagoptaminian — COMMENTED** [review]
@pyphilia (@juancarlosfarah) this looks really nice. left some small/minor comments.
An idea: Since the 'values in x' is just two options (1900 and today) we can do it like the _Waves_-_Flux_ toggle (`SwitchWithTwoLabels`).

File: src/components/common/GreenhouseEffectsSettings.js (hunk 5 above)
Comment (pyphilia) [reply src/components/common/GreenhouseEffectsSettings.js:28]: Not at all^^ The first argument is actually a key, so you can input anything you want. If you check the translations you will see I added the subscript tag directly!

**Pull request merged**